src/test/java/dev/amp/validator/utils/CssSpecUtilsTest.java

```diff
@@ -100,15 +100,15 @@ public Void answer(final InvocationOnMock invocation) {
   public void testExtractUrls() {
 
     final List<Token> tokens = new LinkedList<>();
-    tokens.add(new Token(22, "@media"));
-    tokens.add(new Token(55, "screen"));
-    tokens.add(new Token(1, "\n"));
-    tokens.add(new Token(22, "background-image"));
-    tokens.add(new Token(61, ":"));
-    tokens.add(new Token(100, "https://www.somesite.com/image.jpg"));
-    tokens.add(new Token(60, ";"));
-    tokens.add(new Token(1, "\n"));
-    tokens.add(new Token(0, ""));
+    tokens.add(new Token(SACParserCSS3Constants.IDENT, "@media"));
+    tokens.add(new Token(SACParserCSS3Constants.LBRACE, "screen"));
```

Comment:
55 does = LBRACE, but obviously "screen" is not a {
not sure what this should be, but I kept int value the same when doing this